### PR TITLE
Bogdan/200514/update copy description readme title

### PR DESCRIPTION
**Pull request by bvdr**
- [x] Updated readme file
- [x] Updated email copy
- [x] Updated plugin title
- [x] Bumped plugin version

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -1,37 +1,47 @@
...
+Yes but the plugin works just fine with the default settings.
+For simplicity we'll showcase and explain the available options:
+**Number of IPs to save**
+* 0 is the default setting.
+* By default the plugin stores all login IPs in user history.
+* Any number larger than 0 determines the number of login IPs to save in user history.
 
-Question: Does the plugin stores any data?
-Answer: The plugin tracks the last IP\'s used to login for each account and stores them in a custom table for performance reasons.
+= Does the plugin stores any data? =
```
Comment (mircian) [inline readme.txt]: Does the plugin "store" any data

File: readme.txt [hunk 2]
```diff
@@ -1,37 +1,47 @@
...
+The plugin tracks the last IP's used to login for each account and stores them in a custom table for performance reasons.
 
-Question: Does this plugin is GDPR compliant?
-Answer: This version is not GFPR compliant. The IP\'s are stored plain, un-anonymized. We planed for a GDPR compliance feature soon.
+= Does this plugin is GDPR compliant? =
```
Comment (mircian) [inline readme.txt]: "Is this plugin GDPR compliant"

**Review by mircian — CHANGES_REQUESTED** [review]
Grammar

File: inc/login.php [hunk 3]
```diff
@@ -78,7 +78,10 @@ public function new_ip_email( $user, $ip ) {
 		$email_to   = $user->user_email;
 		$site_title = get_bloginfo( 'name' );
 
-		wp_mail( $email_to, 'New login to your account on ' . $site_title, 'We detected a new login from: ' . $ip . ' - If this was you please ignore this email.' );
+		//Message body
+		$message_body_text = sprintf( esc_html__( 'We detected a new login from: %1$e - If this was you please ignore this email. If you don\'t recognize the login attempt we recommend you to login and change your password.', 'was-it-you' ), $ip );
```
Comment (mircian) [inline inc/login.php]: `%1$e`  should be `%1$s`

File: readme.txt (hunk 1 above)
Comment (bvdr) [reply readme.txt]: Done!

File: readme.txt (hunk 2 above)
Comment (bvdr) [reply readme.txt]: Done!

File: inc/login.php (hunk 3 above)
Comment (bvdr) [reply inc/login.php]: Done

**Review by mircian — APPROVED** [review]

**Pull request merged**